### PR TITLE
Sync support for multiple app filters in STCs

### PR DESCRIPTION
**Pull request by EmanueleGallone**

### CODE REVIEW

File: scenarios/.env.stc-leafspine [hunk 1]
```diff
@@ -34,3 +34,5 @@ P4RT_UP4_METERS=2
 P4RT_UP4_FLOWS_APP_FILTER=2
 # Used in pfcp-* scenarios to check UP4 flows, should be ENODEBS * 5
 PFCP_UP4_FLOWS=5
+PFCP_UP4_SESSIONS=5
```
Comment (daniele-moro) [inline scenarios/.env.stc-leafspine:37]: Can't we use directly `PFCP_UP4_FLOWS`?

File: .env.stable [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 PFCP_AGENT_IMAGE=omecproject/upf-epc-pfcpiface:master-2a4452c
 ONOS_IMAGE=opennetworking/sdfabric-onos:master-2022-03-23
-PFCPSIM_IMAGE=opennetworking/pfcpsim:6be4ce2
+#PFCPSIM_IMAGE=opennetworking/pfcpsim:6be4ce2
+PFCPSIM_IMAGE=pfcpsim:0.2.0-dev
```
Comment (EmanueleGallone) [inline .env.stable:8]: To be changed once changes on pfcpsim are merged

File: scenarios/.env.stc-leafspine [hunk 3]
```diff
@@ -34,3 +34,4 @@ P4RT_UP4_METERS=2
 P4RT_UP4_FLOWS_APP_FILTER=2
 # Used in pfcp-* scenarios to check UP4 flows, should be ENODEBS * 5
 PFCP_UP4_FLOWS=5
+PFCP_UP4_APP_FILTER_FLOWS=15
```
Comment (daniele-moro) [inline scenarios/.env.stc-leafspine:37]: Please, add a comment on how to calculate this value (see above).
Also, we miss this value for the singlepair setup

File: scenarios/.env.stc-leafspine [hunk 4]
```diff
@@ -34,3 +34,5 @@ P4RT_UP4_METERS=2
 P4RT_UP4_FLOWS_APP_FILTER=2
 # Used in pfcp-* scenarios to check UP4 flows, should be ENODEBS * 5
 PFCP_UP4_FLOWS=5
+# Used in pfcp-app-filtering.xml scenario to check UP4 flows. Should be ENODEBS * number of app-filters * 5
```
Comment (daniele-moro) [inline scenarios/.env.stc-leafspine:37]:
```suggestion
# Used in pfcp-app-filtering.xml scenario to check UP4 flows. Should be ENODEBS * 3 (number of app filters) * 5
```

**Comment by daniele-moro** [comment]
retest this please

**Pull request merged**